src/public/sass/pages/_confirmation-page.scss

```diff
@@ -0,0 +1,18 @@
+.print-summary-body {
```

Comment:
Remove and use `.govuk-!-display-none` instead. The point is to reuse as much as we can from GDS styles.